src/table/operations.jl

```diff
@@ -0,0 +1,85 @@
+import Base: filter, map, reduce
+
+"""
+    function map(f, d::DTableRows)
+
+Applies `f` to each row of the `DTable`.
+
+# Examples
+```jldoctest        
+    d = DTable((a=rand(1000), b=rand(1000));chunksize=100)
+    m = map(x -> x.a + x.b, eachrow(d))
+    fetch(m)
+```
+"""
+function map(f, d::DTableRows)
+    chunk_wrap = chunk -> begin 
+        i = eachrow(chunk)
+        map(f, i)    
+    end
```

Comment:
Although these really should be pulled out into dedicated functions, because otherwise we'll incur a lot of compile time (due to specializing the closure object on the captured `f`).